web/react/components/sidebar_right_menu.jsx

```diff
@@ -6,6 +6,10 @@ var client = require('../utils/client.jsx');
 var utils = require('../utils/utils.jsx');
 
 export default class SidebarRightMenu extends React.Component {
+    componentDidMount() {
+        $('.sidebar--left .dropdown-menu').perfectScrollbar();
```

Comment:
We are but unless it is removed we'll try to provide a better experience as much as we can. Once we implement a react scrollbar we can change this one with that.
Also we didn't have a perfectscrollbar in the dropdown-menu before.